README.md

```diff
@@ -4,30 +4,51 @@
 [![Build Status](https://travis-ci.org/ExchangeUnion/xud.svg?branch=master)](https://travis-ci.org/ExchangeUnion/xud)
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/87238907485845eb879bd61c65561775)](https://www.codacy.com/app/sangaman/xud?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=ExchangeUnion/xud&amp;utm_campaign=Badge_Grade)
 
-[Exchange Union](https://www.exchangeunion.com/) (XU) is a decentralized exchange layer built on the [Lightning](http://lightning.network/) and [Raiden](https://raiden.network/) networks to enable trustless and instant cryptocurrency swaps and order fulfillment between exchanges.
-
-This repo contains the early stages of the Exchange Union Daemon (`xud`) which encompasses the following components:
+[Exchange Union](https://www.exchangeunion.com/) (XU) is a decentralized exchange layer built on the [Lightning](http://lightning.network/) and [Raiden](https://raiden.network/) networks to enable trustless and instant cryptocurrency swaps and order fulfillment between exchanges. This repository contains the Exchange Union Daemon (`xud`) software which encompasses the following components:
 
 * Integration with [lnd](https://github.com/lightningnetwork/lnd) and [raiden](https://github.com/raiden-network/raiden) nodes.
-* Order book data stored in a local mysql/mariadb database.
-* Peer-to-peer networking with other XU nodes
-* A gRPC API with web proxy to serve other applications, also accessible by command-line interface.
+* Decentralized order book to aggregate orders from the known network.
+* Matching engine to match incoming local orders with existing local and remote orders and initiate atomic swaps.
+* Peer-to-peer networking with other XU nodes.
+* gRPC API with web proxy to serve other applications, also accessible by command-line interface.
+
+## Install
+
+If you already have [Node.js](https://nodejs.org/en/download/) installed, you can install `xud` via npm by running
+
+```bash
+sudo npm install xud -g --unsafe-perm
+```
 
-Contributions, feedback, and questions are welcome.
+Detailed instructions for installing `xud` can be found in the wiki on our [Installation](https://github.com/ExchangeUnion/xud/wiki/Installation) page. Instructions for installing via Docker are on our [Docker](https://github.com/ExchangeUnion/xud/wiki/Installation) page.
```

Comment:
The Link to the Docker page is [`https://github.com/ExchangeUnion/xud/wiki/Docker`](https://github.com/ExchangeUnion/xud/wiki/Docker). And on that page you missed a space between the `#` and the title itself.